engines/bastion_katello/app/assets/javascripts/bastion_katello/products/products.controller.js

```diff
@@ -47,10 +47,16 @@ angular.module('Bastion.products').controller('ProductsController',
             'paged': true
         };
 
-        nutupane = new Nutupane(Product, params);
+        nutupaneParams = {
+            'disableAutoLoad': true
+        };
+        $scope.disableRepoDiscovery = true;
+        nutupane = new Nutupane(Product, params, undefined, nutupaneParams);
         $scope.controllerName = 'katello_products';
         nutupane.masterOnly = true;
-
+        nutupane.refresh().then(function () {
```

Comment:
Sorry I didn't catch this the first time I looked, but I think this is a behavior worth having a test for - can you add one here?

Reply:
@jturel: Added some tests. Also, some linting that Rubymine did for me.